### PR TITLE
Add helper methods to parse and set the GeneralConfiguration node

### LINKED ISSUE #1: Set helper methods to parse the GeneralConfiguration YAML node
- [x] parser helper methods
- [x] setting the appropriate class variables in the parser

### PR DESCRIPTION
**Pull request by UCLGuichard**
Fixes #1

### CODE REVIEW

File: src/yaml-parser.cpp [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  void parseAndSetInput(YAML::Node node);
+  // Variables
+  std::string filename;
+  std::string logging;
+  int iterations;
+  int epsilonScaling;
+  std::string gamma;
+  std::string output_prefix;
+  std::string skymodel;
+  std::string measurements;
```
Comment (Luke-Pratley) [inline src/yaml-parser.cpp]: Gamma should be double. Epsilon scaling should be double.

Comment (UCLGuichard) [reply src/yaml-parser.cpp]: done

**Comment by UCLGuichard** [comment]
Sure. Let's discuss it when you're there. Another thing `gamma` is set to `default` in the `config.yaml`. Should it be a double then ?

**Comment by UCLGuichard** [comment]
Merging now.

**Pull request merged**